### PR TITLE
Gradle'zing card-applets

### PR DESCRIPTION
**Pull request by typelogic**
Following status-keycard's `com.fidesmo:gradle-javacard:0.2.7`.

### CODE REVIEW

**Review by jeremi — CHANGES_REQUESTED** [review]
- Can you update the Readme with the build information?
- cleanup the build.sh

File: build.gradle.auth [hunk 1]
```diff
@@ -0,0 +1,66 @@
+group 'card-applets'
```
Comment (jeremi) [inline build.gradle.auth]: Move this file to /auth/build.gradle

File: build.gradle.sam [hunk 2]
```diff
@@ -0,0 +1,65 @@
+group 'card-applets'
```
Comment (jeremi) [inline build.gradle.sam]: Move this file to /sam/build.gradle

File: build.gradle.tools [hunk 3]
```diff
@@ -0,0 +1,59 @@
+group 'card-applets'
```
Comment (jeremi) [inline build.gradle.tools]: Move this file to /tools/build.gradle

**Review by jeremi — COMMENTED** [review]
- After moving the gradle configs, create gradle.config in the root directory that will call the build of each applets using dependencies ( https://discuss.gradle.org/t/how-to-invoke-subtasks-of-subprojects-in-the-parent-build-gradle/6622 ) so in the root, we can type `./gradlew build` to build all the cap files

**Comment by typelogic** [comment]
https://github.com/gradle/gradle/issues/1461 is regarding the `_JAVA_OPTIONS` . I have separate PRs in the respective repo for `tools`, `auth`, and `sam`, in combination to this one.

File: build.gradle.auth (hunk 1 above)
Comment (typelogic) [reply build.gradle.auth]: A PR has been submitted to the `card-auth-applet` repo

File: build.gradle.sam (hunk 2 above)
Comment (typelogic) [reply build.gradle.sam]: A PR has been submitted to the `card-sam-applet` repo.

File: build.gradle.tools (hunk 3 above)
Comment (typelogic) [reply build.gradle.tools]: A PR has been submitted to the `card-tools-applet` repo.

**Pull request merged**